code/modules/jobs/job_types/roguetown/adventurer/types/sentinel.dm

```diff
@@ -0,0 +1,50 @@
+/datum/subclass/sentinel
```

Comment:
I don't think we need the unique class like sentinel involved. rare classes from old age of adventurers had a % chance to roll, and this is meant to be a migrant wave. Can probably be removed.